cmd/run.go

```diff
@@ -245,7 +245,7 @@ func init() {
 	RunCmd.Flags().StringVarP(&logLevel, "log-level", "", "info", "Sets the log verbosity. Valid values are: debug, info, warn, error, fatal, or panic. Default is info")
 	RunCmd.Flags().IntVarP(&maxConcurrency, "max-concurrency", "", -1, "controls the concurrency level of the app. Default is unlimited")
 	RunCmd.Flags().StringVarP(&protocol, "protocol", "", "http", "tells Dapr to use HTTP or gRPC to talk to the app. Default is http")
-	RunCmd.Flags().StringVarP(&componentsPath, "components-path", "", "", "Path for components directory. Default is ./components.")
+	RunCmd.Flags().StringVarP(&componentsPath, "components-path", "", "", "Path for components directory. Default is ~/.dapr/components or %USERPROFILE%\\.dapr\\components")
```

Comment:
Not an issue but to be more clear we can say :

"Default is ~/.dapr/components for linux and %USERPROFILE%\\.dapr\\components for windows"

Reply:
I thought about it but then we would need to explain macOS too.